src/components/dropdowns/CynDropdown.vue

```diff
@@ -1,40 +1,34 @@
 <template>
-    <div id="dropdown-div">
-        <label for="users">Select Users:</label>
-        <br />
-        <select v-model="selected">
-            <option value disabled>Please Select a User</option>
-            <option v-bind:value="user.name" v-for="user in users" :key="user.id"
-                >{{ user.name }} | ID: {{ user.id }}</option
-            >
-        </select>
-        <p>Current user selected: {{ selected }}</p>
-    </div>
+  <div id="dropdown-div">
+    <label for="users">Select an Item:</label>
+    <br />
+    <select @change="onChange($event)">
+      <option value disabled selected>Please Select an item</option>
+      <option
+        v-for="item in items"
+        :value="item.name"
+        :key="item.id"
+      >{{ item.name }} | ID: {{ item.id }}</option>
```

Comment:
this line that starts off with a `>` is a real eyesore for me. since we don't have a convention in place for this yet, it's up to you for now. if you don't have a preference i'd personally prefer this as one line.